install/kubernetes/helm/istio/charts/sidecar-injector/templates/deployment.yaml

```diff
@@ -37,6 +39,24 @@ spec:
           - name: inject-config
             mountPath: /etc/istio/inject
             readOnly: true
+          livenessProbe:
+            exec:
+              command:
+                - /usr/local/bin/sidecar-injector
+                - probe
+                - --probe-path=/health
```

Comment:
Are you sure /health is the file path?

Reply:
`/health` is the path that is configured on line 31 above. I'm only using the `probe` package for checking the file modification time. The file write/update logic is specific to the injector so I could incorporate it into the simple run loop.

Reply:
I actually meant the file path could be /tmp/health instead of /health. The process may need root access to write files on root directory.